FSJ_django20_project/FSJ/models_student.py

```diff
@@ -10,6 +10,7 @@
 class Student(FSJUser):
     program = models.ForeignKey(Program, on_delete = models.SET_NULL, null = True, blank = True, verbose_name = _("Program"))
     year = models.ForeignKey(YearOfStudy, on_delete=models.PROTECT, verbose_name = _("Year"))
+    gpa = models.CharField(max_length = 255, null = True, blank = True, verbose_name = _("GPA"))
```

Comment:
GPA is a char? Shouldn't it be a float?

Reply:
The GPAs in the file they provided included high school marks, which are in the form of percentages rather than numbers ("88%" as opposed to "3.8"). Made it a charfield to accommodate both.

Reply:
Understood, works for me. Maybe max length 255 is extreme though?

Reply:
Fair point. Do you think length 10 is enough? Can't imagine they'd have GPAs longer than that.